src/_blog/SRI-1.0.0.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Introducing SRI 1.0.0 - the most important milestone in our Stratum V2 journey!"
+description: "We are thrilled to announce the launch of SRI 1.0.0! This marks an important milestone in our mission to decentralize and enhance bitcoin mining and stands ready for immediate testing and integration."
+date: "2024-20-04"
+authors:
+  - Pavlenex
+tags:
+  - StratumV2
+  - SRI
+  - Release
+
+---
+
+We are thrilled to announce the launch of SRI [1.0.0](https://github.com/stratum-mining/stratum/releases)! This marks an important milestone in our mission to decentralize and enhance bitcoin mining and stands ready for immediate testing and integration.
+
+Start your SV2 journey with our [getting started guide](https://stratumprotocol.org/getting-started/).  
+
+## 🛎️ Key highlights
+
+### 👷 Miners  
+- Connect to a SV2 pool without upgrading existing SV1 firmware through Translation Proxy or
+- Connect to an SV2 pool directly with an SV2 firmware device (BraiinsOS)
+- Run you own bitcoin, construct templates, and declare them to the SV2 Pool
+- Utilize pool fallback functionality in case the pool decides to censor
+
+### 🛠️ Pools
+- Deploy and operate a very simple SV2 Pool 
+- Integrate SV2 into an existing pool through our libraries
+
+The new version is a result of improvements in the Stratum V2 [specification](https://github.com/stratum-mining/sv2-spec) through the working [group](https://www.cnbc.com/2022/10/11/bitcoin-mining-software-overhaul-stratum-v2-promoted-by-block-braiins.html) collaboration and rigorous testing with our interoperability suite to ensure compatibility between the two implementations of the Stratum V2 protocol. 
+
+As we planned [eight months ago](https://stratumprotocol.org/blog/sri-roadmap-2023/), the release allows miners running SV1 or SV2 firmware to connect to an SV2 Pool (directly or through a translation proxy), optionally using a locally hosted Job Declaration Client and a patched version of the Bitcoin Node to **construct their own block templates** which are then sent to the Job Declarator Server run by the SV2 Pool. It’s important to mention that our code refactoring introduced the ability for the node and JDC to be run by an independent third-party, not just the miners.
```

Comment:
```suggestion
As we planned [eight months ago](https://stratumprotocol.org/blog/sri-roadmap-2023/), the release allows miners running SV1 or SV2 firmware to connect to an SV2 Pool (directly or through a translation proxy), optionally using a locally hosted Job Declaration Client and a patched version of a Bitcoin Core Node to **construct their own block templates** which are then sent to the Job Declarator Server run by the SV2 Pool. It’s important to mention that our code refactoring introduced the ability for the node and JDC to be run by an independent third-party, not just the miners.
```

Because you specify the Bitcoin node is patched, it may be worth clarifying that it has to be a Bitcoin Core node.